internal/database/postgres.go

```diff
@@ -0,0 +1,17 @@
+package database
+
+import (
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"os"
+	"vitess.io/vitess/go/vt/log"
```

Comment:
RevSuit 统一使用的是 clog 作为日志库。
```suggestion
	log "unknwon.dev/clog/v2"
```